diff --git a/ps2xRecomp/CMakeLists.txt b/ps2xRecomp/CMakeLists.txt
index 857102a..5ed0d07 100644
--- a/ps2xRecomp/CMakeLists.txt
+++ b/ps2xRecomp/CMakeLists.txt
@@ -130,7 +130,7 @@ install(DIRECTORY include/
     DESTINATION include
 )
 
-include("${CMAKE_SOURCE_DIR}/ps2xRuntime/cmake/ReleaseMode.cmake")
+include("${CMAKE_CURRENT_SOURCE_DIR}/../ps2xRuntime/cmake/ReleaseMode.cmake")
 
 if(CMAKE_BUILD_TYPE STREQUAL "Release" OR CMAKE_BUILD_TYPE STREQUAL "RelWithDebInfo")
     EnableFastReleaseMode(ps2_recomp_lib)
